Remotion/Web/Development.WebTesting/WebTestConfigurationFactory.cs

```diff
@@ -65,6 +65,10 @@ public class WebTestConfigurationFactory
     public IBrowserConfiguration CreateBrowserConfiguration ()
     {
       var configSettings = WebTestConfigurationSection.Current;
+
+      if (configSettings.BrowserName == "Edge")

```

Comment:
I understand this change but it's a little weird that checking for Edge is different than checking for all other browsers. I would suggest to either add a comment why we're doing it this way or change the other browser checks to something that resembles this.